store/pruning/manager_test.go

```diff
@@ -124,14 +168,23 @@ func TestStrategies(t *testing.T) {
 				default:
 					if curHeight > int64(curKeepRecent) && curHeight%int64(curStrategy.Interval) == 0 {
 						pruningHeightExpected := curHeight - int64(curKeepRecent) - 1
-						if tc.snapshotInterval > 0 && snHeight < pruningHeightExpected {
+						if curSnapshotInterval > 0 && snHeight < pruningHeightExpected {
 							pruningHeightExpected = snHeight
 						}
 						require.Equal(t, pruningHeightExpected, pruningHeightActual, curHeightStr)
 					} else {
 						require.Equal(t, int64(0), pruningHeightActual, curHeightStr)
 					}
 				}
+
+				// Change snapshot interval after some height
+				if curHeight == 55000 {
+					curSnapshotInterval = tc.secondSnapshotInterval
+					if int64(curSnapshotInterval-1) > snHeight {
+						snHeight = int64(curSnapshotInterval - 1)
+					}
+					manager.SetSnapshotInterval(tc.secondSnapshotInterval)
+				}
```

Comment:
where does 55000 come from?

Reply:
The test simulates the height from 0 to 110000. So, I chose 55000 (half of 110000) as the changing interval height.